tests/checks/test_stablereq.py

```diff
@@ -34,6 +46,21 @@ def _setup(self, tmp_path, tool, make_repo, make_git_repo):
         self.child_git_repo.run(['git', 'remote', 'set-head', 'origin', 'main'])
         self.child_repo = make_repo(self.child_git_repo.path)
 
+        self.stable_time = self._stable_time_params_default["days"]
+        self.less_days = self._stable_time_params_default["less"]
+        self.more_days = self._stable_time_params_default["more"]
+
+    @pytest.fixture(
+        params=_stable_time_params,
+        ids=map(lambda paramset: f"stable_time={paramset['days'] or 'unset'}", _stable_time_params)
+    )
+    def stable_time(self, request):
+        self.stable_time = request.param["days"]
+        self.less_days = request.param["less"]
+        self.more_days = request.param["more"]
+
+        return request.param
+
     def init_check(self, options=None, future=0):
```

Comment:
Let's pass the `stable_time` as an argument here, so less using of `self.`:

```suggestion
    def init_check(self, options=None, future=0, stable_time=None):
```

And then change the `if self.stable_time is not None:` to use the argument directly

Reply:
This makes much more sense, I'll implement this.

Reply:
Implemented in [16aeb72](https://github.com/pkgcore/pkgcheck/pull/429/commits/16aeb7281e82bb7ae0801f7589d0ce740c7ca44f)